Add missing cstdint

ParticipantDescription.hpp:32:23: error: 'uint64_t' does not name a type
   32 | using ParticipantId = uint64_t;                                                                                                                                                                                |                       ^~~~~~~~
ParticipantDescription.hpp:25:1: note: 'uint64_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'

Upstream-Status: Pending

Signed-off-by: Rob Woolley <rob.woolley@windriver.com>

Index: rmf-traffic-3.5.0-1/include/rmf_traffic/schedule/ParticipantDescription.hpp
===================================================================
--- rmf-traffic-3.5.0-1.orig/include/rmf_traffic/schedule/ParticipantDescription.hpp
+++ rmf-traffic-3.5.0-1/include/rmf_traffic/schedule/ParticipantDescription.hpp
@@ -18,6 +18,7 @@
 #ifndef RMF_TRAFFIC__SCHEDULE__PARTICIPANTDESCRIPTION_HPP
 #define RMF_TRAFFIC__SCHEDULE__PARTICIPANTDESCRIPTION_HPP
 
+#include <cstdint>
 #include <string>
 #include <unordered_map>
 
